Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/StubIrMetadataEmitter.kt

```diff
@@ -560,4 +565,24 @@ private class MappingExtensions(
 
     private val TypeParameterStub.id: Int
         get() = typeParametersInterner.intern(this)
+}
+
+private enum class TypeAliasExpansionMode {
+    /**
+     * Substitution of type alias with fully expanded type.
+     * Typical usage: expanded type of typealias.
+     */
+    ERASE,
```

Comment:
Do I understand correctly that this is the only new mode?
`NONE` is equal to `shouldExpandTypeAliases = false`, while `EXPANSION` is equal to `shouldExpandTypeAliases  = true`?

Reply:
Yep.